example/geny-window.js

```diff
@@ -155,6 +155,8 @@ const fetchRecipes = async () => {
         }
 
         const selectElement = document.querySelector('#listRecipes');
+        // Remove our "warning" select option about an invalid/missing api token.
+        selectElement.remove(0);
         const placeholderOption = document.createElement('option');
```

Comment:
well play, but the bug is worth that what you detect. Every time the input apitoken loose focus, the recipes list is added to the previous.

This will fix all issues
```
const selectElement = document.querySelector('#listRecipes');
     // Flush select before adding new options
      selectElement.innerHTML = '';
 const placeholderOption = document.createElement('option');
```

Reply:
> well play, but the bug is worth that what you detect. Every time the input apitoken loose focus, the recipes list is added to the previous.

Ah, nice catch! 😅 

I've adjusted the fix with your suggestion. Thanks!